mappings/net/minecraft/client/gl/GlFramebuffer.mapping

```diff
@@ -0,0 +1,43 @@
+CLASS none/bpf net/minecraft/client/gl/GlFramebuffer
+	FIELD a texWidth I
+	FIELD b texHeight I
+	FIELD c viewWidth I
+	FIELD d viewHeight I
+	FIELD e useDepthAttachment Z
+	FIELD f fbo I
+	FIELD g colorAttachment0 I
```

Comment:
In that case, we can always rename, and we should always be willing to. That's just my opinion.